src/compose/ComposeText.js

```diff
@@ -56,17 +56,16 @@ class ComposeText extends React.Component {
       contentHeight: MIN_HEIGHT,
     };
   }
-
   handleSend = () => {
     const { auth, narrow } = this.props;
     const { text } = this.state;
-
+    const message = replaceEmojiWithTextCode(text);
```

Comment:
I have done this because, until these emoji's are registered at backend we can still show them on our application.